package.json

```diff
@@ -64,12 +62,18 @@
     "gulp-util": "^3.0.7",
     "istanbul": "^0.4.3",
     "jsdom": "^9.12.0",
-    "lcov-result-merger": "^1.2.0",
     "mocha": "^3.2.0",
     "npm-run-posix-or-windows": "^2.0.2",
+    "typescript": "^2.6.1",
     "zone.js": "^0.7.3"
   },
   "dependencies": {
+    "@types/configstore": "^2.1.1",
```

Comment:
how many of these types do we expose in our public facing definitions? feels like a lot could be moved to devDependencies

Reply:
> how many of these types do we expose in our public facing definitions?

I don't actually know how this works, so happy to move them.

If it's a transitive dep, does the compiler not need types for those as it walks dependencies? Or does it stop at the `node_modules` border and just checks use of the public interface of the module? (most of the `@types/*` we use don't have deps themselves, but some do...e.g. `inquirer` pulls in typings for `through` and a bunch of `rx` modules)

Reply:
my understanding is that you need to include any types you expose in your `.d.ts` files as dependencies so when consumers try to compile they'll be able to find them.

the only case where you need to include internal type dependencies is when you expect your consumers to also be compiling your ts themselves, not just consuming the type definitions and js. since we have no ts to be compiling, it seems like we should be ok here?

Reply:
[this comment](https://github.com/Microsoft/types-publisher/issues/81#issuecomment-234058057) is what led me to believe this

Reply:
cool, I'll do that then. I can't think of anywhere in our module API where we expose or depend on external types